.github/workflows/build-repo.yml

```diff
@@ -0,0 +1,97 @@
+name: Release workflow
+on:
+  workflow_run:
+    workflows: ['lint-test']
+    branches: [main, ci-beta, qa-beta, stage-beta, prod-stable, ci-stable, qa-stable, stage-stable, prod-stable]
```

Comment:
I don't think all of these branches will be relevant to us, I would like to see us using more than just `main` branch, but right now that's all we have.